docs/source/receptorctl/receptorctl_work_cancel.rst

```diff
@@ -0,0 +1,26 @@
+-----------
+work cancel
+-----------
+
+.. contents::
+   :local:
+
+``receptorctl work cancel`` cancels (kills) one or more units of work.
```

Comment:
```suggestion
``receptorctl work cancel`` kills one or more units of work.
```

Reply:
I'd say it's better not to repeat the word "cancel" here. Consider "stops" or "terminates" as an alternative to "kills" - but I think it's fine given there is the standard `kill` command.

In the work unit docs the `canceled` state uses the word "terminate": https://github.com/ansible/receptor/blob/devel/docs/source/receptorctl/receptorctl_work_list.rst#work-unit-states